charts/div-rfe/requirements.yaml

```
@@ -1,14 +1,14 @@
 dependencies:
   - name: nodejs
-    version: ~1.8.0
+    version: 2.0.0
     repository: '@hmctspublic'
   - name: redis
     version: 10.5.7
     repository: '@stable'
     tags:
       - pr
   - name: idam-pr
-    version: ~2.1.3
+    version: ~2.2.6
```

we have a tool that can automatically update these but it doesn't work with ~ so I would recommend removing it
```suggestion
    version: 2.2.6
```

---

Done